scripts/gen-i18n.js

```diff
@@ -32,7 +32,15 @@ const walk = require('walk');
 const parser = require("@babel/parser");
 const traverse = require("@babel/traverse");
 
-const TRANSLATIONS_FUNCS = ['_t', '_td'];
+// Find the package.json for the project we're running gen-18n against
+const projectPackageJsonPath = path.join(process.cwd(), 'package.json');
```

Comment:
This isn't really the safest way to get at the package.json, but should be fine I think. There's still zero stable way to get at the thing in JS :(

Reply:
Yeah, wish there is was a nice canonical solid way. I tested this and works as expected for our setups.